v2/pkg/commands/build/build.go

```diff
@@ -151,6 +165,51 @@ func Build(options *Options) (string, error) {
 	return compileBinary, nil
 }
 
+func generateBindings(options *Options) error {
+	options.Logger.Print("  - Generating obfuscated bindings: ")
+	defer func() {
+		options.Logger.Println("Done.")
+	}()
+	// Run go mod tidy to ensure we're up-to-date
+	stdout, stderr, err := shell.RunCommand(".", "go", "mod", "tidy")
+	if err != nil {
+		options.Logger.Println(stdout)
+		options.Logger.Println(stderr)
+		return err
+	}
+
+	self := os.Args[0]
+	var env []string
+	if options.Obfuscate {
+		env = append(env, "WAILS_OBFUSCATE=true")
+	}
+
+	if len(options.UserTags) > 0 {
+		genModuleTags := lo.Without(options.UserTags, "desktop", "production", "debug")
+		genModuleTagsString := strings.Join(genModuleTags, ",")
+		err = runCommandWithEnv(".", true, env, self, "generate", "module", "-tags", genModuleTagsString)
+	} else {
+		err = runCommandWithEnv(".", true, env, self, "generate", "module")
+	}
+	return err
+}
+
+func runCommandWithEnv(dir string, exitOnError bool, env []string, command string, args ...string) error {
+	cmd := exec.Command(command, args...)
+	cmd.Dir = dir
+	cmd.Env = append(os.Environ(), env...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		println(string(output))
+		println(err.Error())
+		if exitOnError {
```

Comment:
Should we let the caller decide if he wants to exit or not depending on the error return value?